### DISCUSSION TITLE
thanking to github

### DISCUSSION THREAD

**Discussion by gaurav4569** [community/community#19426] [Other Feature Feedback, Questions, & Ideas]
thanks github for the badge

**Comment by melvincwng** [comment]
@gaurav4569, congratulations on your Pull Shark achievement badge! Looking good on you 🎉

Just FYI, if you want to know a full list of all possible achievements that can be obtained on Github, check out [this page](https://github.com/Schweinepriester/github-profile-achievements).

Otherwise, welcome to GitHub Community 🥳!

**Reply by abdulwahidkahar** [reply]
congratulations on your Pull Shark achievement badge! Looking good on you 🎉

**Reply by gaurav4569** [reply]
thanks you so much

**Comment by gaurav4569** [comment]
https://discord.gg/5kyBPMgTF2 pls join this server

**Comment by itsmeshibintmz** [comment]
Thanks GitHub for the Badges. Really love how it colours my profile. It's an inspiration to contribute more. Loving it.

**Reply by gaurav4569** [reply]
yes u are right
